esmvaltool/diag_scripts/extreme_events/make_Glecker_plot2.r

```diff
@@ -0,0 +1,663 @@
+# #########################################################################################################
+# make_Glecker_plot2.r
+#
+# Author: Christian W. Mohr (CICERO, Norway)
+#         Marit Sandstad (CICERO, Norway)
+#
+#
+# #########################################################################################################
+# Description:
+# Code to plot Glecker polygon diagram to compare climdex index
+# 	performance between models and reanalysis.
+#
+# Modification history
+#
+#    2019 0506-hard_jo: conversion to ESMValTool2
+#    20180601-A_cwmohr: re-creation (complete new script incorparating segments from "make_timeseries_plot.r & make_Glecker_plot.r")
+#
+# #########################################################################################################
+
+gleckler_main <- function(path = "./",
+                          idx_list = c(
+                            "tn10pETCCDI_yr", "tn90pETCCDI_yr",
+                            "tx10pETCCDI_yr", "tx90pETCCDI_yr",
+                            "tnnETCCDI_yr", "tnxETCCDI_yr",
+                            "txnETCCDI_yr", "txxETCCDI_yr"
+                          ),
+                          model_list = c(
+                            "EC-EARTH", "HadGEM2-ES",
```

Comment:
Having these models hardcoded here is not such a great idea

Reply:
yes, removed